src/UsblParser.cpp

```diff
@@ -410,8 +410,6 @@ DeviceSettings UsblParser::parseCurrentSettings (string const &buffer)
             settings.carrierWaveformId = atoi(splitted2.at(1).c_str());
         else if(splitted2.at(0).find("Local Address") != string::npos)
```

Comment:
It could be, but I'm not sure how many digits the numbers can have or if they can vary.
I implemented the second option, once I know how to split the message and with text should be present.